### DISCUSSION TITLE
Limit subfolders depth

### DISCUSSION THREAD

**Discussion by stefanKBL** [Studio-42/elFinder#3332] [Q&A]
Hi, is there a way to limit the subfolders creation depth. eg
- root
    - thenFolderA
    - thenFolderB
    - ...
But do not accept creation under FolderA.
I try to use bind in backend constructor with mkdir.pre, but not able to check if parent dir is root or not.
Thanks

**Comment by nao-pon** [comment]
You can verify that the parent directory is root as follows:

```php
$target = $volume->file($args['target']);
if (!empty($target['phash']) && strpos($target['phash'], $volume->id()) === 0) {
    // not root
    $parent = $volume->file($args['phash']);
} else {
    // is root
}
```
